worker.py

```
@@ -0,0 +1,195 @@
+                # Dequeue next email (blocking with timeout)
+                email_id = await queue_service.dequeue()
+                
+                if email_id:
+                    # Process email
+                    async with AsyncSessionLocal() as db:
+                        await process_email(db, email_id)
+                        
+            except asyncio.CancelledError:
+                logger.info("Worker task cancelled")
+                break
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)